modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectAction/tabs/ActionBuilder.tsx

```diff
@@ -199,6 +199,23 @@ export default function ActionBuilder({
 
 	return (
 		<>
+			{disableGroovyAction && (
+				<ClayAlert
+					className="lfr-objects__side-panel-content-container"
+					displayType="warning"
+					title="Warning"
+				>
+					{Liferay.Language.get(
+						'groovy-script-validations-are-deactivated-for-your-instance'
+					)}{' '}
+					<a>
+						{Liferay.Language.get(
+							'go-to-system-settings-security-tools-script-management-to-allow-administrators-to-execute-groovy-code'
+						)}
+					</a>
```

Comment:
Hey @danvb112, will you add the link to this anchor in the future?

Reply:
Hey @igor-franca , this link will be provided by the backend soon, I will probably add this link in another task